tests/destructor/tdestructor_manual_invocation.nim

```diff
@@ -0,0 +1,32 @@
+discard """
+  matrix: "--gc:refc; --gc:arc"
+  output: '''
```

Comment:
Didn't manage to remove these echos here. 
Could make =destroy push to a seq , and check its status later ; 
But then, need to make the =destroy get called by the runtime - is this possible by adding scopes?

Reply:
You could wrap the top-level statements in a function in order to have the destructors called in a deterministic way. That'd be a change in semantics, but it shouldn't interfere with what the test is testing.

> Could make =destroy push to a seq , and check its status later ;

In most cases, that's the way to go 👍